doc/api/fs.markdown

```diff
@@ -253,6 +253,8 @@ Any specified file descriptor has to have been opened for appending.
 
 _Note: Specified file descriptors will not be closed automatically._
 
+_Note: Do not use `appendFile` to write to a single file from multiple places in an application.  It will open a file descriptor *per call*, not one *per file*.  (Non-specified) file descriptors are closed automatically, but will stay open for milliseconds on a busy host.  Under non-trivial load, the application will quickly exceed the number of file descriptors allowed by the OS, causing seemingly unrelated file and network failures. 
```

Comment:
what's special about multiple places? the problem described can occur if you use this from one place, what you are describing seems overly specific to your application. I think it would be better to say something like "Allocates a file descriptor briefly for each call. This is not as efficient as opening the file once in "append mode", in that it continually opens and closes a fd for (possibly) a single file, and if called in a heavily parallel application may even cause the process wide descriptor limits to be hit."